app/grandchallenge/challenges/views.py

```diff
@@ -368,4 +371,24 @@ def form_valid(self, form):
             send_challenge_status_update_email(
                 challengerequest=form.instance, challenge=challenge
             )
+
+        response = HttpResponse()
+        response["HX-Refresh"] = "true"
+        return response
+
+
+class ChallengeRequestBudgetUpdate(
```

Comment:
Sorry forgot this comment, but is there a test for this view? Both for permissions and functionality.